player/lua/console.lua

```diff
@@ -363,62 +363,25 @@ local function populate_log_with_matches()
 
     local max_log_lines = calculate_max_log_lines()
 
-    if selected_match < first_match_to_print then
-        first_match_to_print = selected_match
-    end
+    if #matches > max_log_lines then
+        log[1] = {
+            text = selected_match .. '/' .. #matches,
```

Comment:
Maybe also show the range of items that we currently have on screen? Like `[${first_visible_idx}, ${last_visible_idx}]` maybe with smaller text too.

Reply:
Added, but I should probably try to implement a scrollbar later.